src/mixins/allowable-actions-mixin.ts

```diff
@@ -69,14 +72,16 @@ export default class AllowableActionsMixin extends Vue {
         return (ff && isBusiness)
       }
 
+      case AllowableActions.CONSENT_AMALGAMATION_OUT: {
+        return this.isAllowedFiling(FilingTypes.CONSENT_AMALGAMATION_OUT)
+      }
+
       case AllowableActions.CONSENT_CONTINUATION_OUT: {
-        const ff = !!GetFeatureFlag('supported-consent-continuation-out-entities')?.includes(this.getLegalType)
-        return (ff && this.isAllowedFiling(FilingTypes.CONSENT_CONTINUATION_OUT))
+        return this.isAllowedFiling(FilingTypes.CONSENT_CONTINUATION_OUT)
       }
```

Comment:
For this and Continuation Out below, I moved the FF check to the menu item itself. This de-couples the FF from the allowable actions so that we correctly hide the menu item if the FF is disabled.